components/HTMLContent.js

```diff
@@ -39,10 +39,12 @@ const ReadFullLink = styled.a`
 
 const InlineDisplayBox = styled.div`
   overflow-y: hidden;
+  max-height: ${props => props.maxHeight + 20}px;
```

Comment:
This prop is not set in most places. We don't want `max-height: undefinedpx` to be happened everywhere. Let's rely on the standard `maxHeight` from styled-system:

> import { maxHeight } from 'styled-system'

```suggestion
  ${maxHeight}
```

Reply:
@Betree : Since we need to specify the `maxHeight` at some places shouldn't we add a conditional instead? I've done it at, 

https://github.com/opencollective/opencollective-frontend/pull/7365/files#diff-7c66bed1a72345f8e57ccf8d28acc894e870d12efc4045b9ab242b0a386b064cR42-R47